packages/library/README.md

```diff
@@ -205,9 +180,9 @@ const rpc = createSolanaRpc('http://127.0.0.1:8899');
 const slot = await rpc.getSlot().send();
 ```
 
-### Using Custom RPC Transports
+### Custom RPC Transports
 
-The `createSolanaRpc` function communicates with the RPC server using a default HTTP transport that should satisfy most use cases. However, you may provide your own transport or decorate existing ones to communicate with RPC servers in any way you see fit. In the example below, we explicitly create a transport and use it to create a new RPC client via the `createSolanaRpcFromTransport` function.
+The `createSolanaRpc` function communicates with the RPC server using a default HTTP transport that should satisfy most use cases. You can provide your own transport or decorate an existing one to communicate with RPC servers in any way you see fit. In the example below, we explicitly create a transport and use it to create a new RPC client via the `createSolanaRpcFromTransport` function.
```

Comment:
Nitpick: I'm not sure quite what "decorate an existing one" means, given we don't have decorators. I wonder if "wrap an existing one" would be a bit clearer.